extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -262,6 +263,33 @@ export class AmpStoryPage extends AMP.BaseElement {
     );
   }
 
+  /**
+   * @private
+   * @return {Element}
+   */
+  maybeConvertCtaLayerToPageOutlink_() {
```

Comment:
Should this logic be moved to [`amp-story-cta-layer`](https://github.com/ampproject/amphtml/blob/main/extensions/amp-story/1.0/amp-story-cta-layer.js)?
If we don't include if there, I think we should remove the `amp-story-cta-layer` code.

Reply:
+1 to removing `amp-story-cta-layer.js`, but keeping it valid AMP, and keep its CSS.

Reply:
Done.